.github/workflows/catalyst.yaml

```diff
@@ -9,9 +9,6 @@ on:
       - "Dockerfile"
       - "docker-compose.yml"
       - "Makefile"
-    branches:
-      - "main"
-      - "*catalyst*"
```

Comment:
@hjpotter92 Any harm in this?

Reply:
i'd leave `main` here, and remove only the `*catalyst*` match. the builds would be triggered for all pull request, the additional `branches` filter applies to direct push to these branches (so we can still have branches which don't necessarily need these builds triggered).